runtime/testy/stateManager.py

```diff
@@ -129,6 +129,7 @@ def send_ansible(self):
 
   def recv_ansible(self, new_data):
     self.state.update(new_data)
+    print(new_data)
```

Comment:
this will mess up testing. do you really need to print, or was this only for debug purposes?